gitlab_runner_config.py

```diff
@@ -44,7 +47,23 @@ def identifying_tags(instance):
     return list(identifiers)
 
 
-def generate_tags(instance, executor_type="", env=None):
+def flatten_values(d):
+    #recursively collect dictionary values into a list
```

Comment:
Make this a string, not a comment since it is a [docstring](https://www.python.org/dev/peps/pep-0257)